docs/cargo.md

```diff
@@ -0,0 +1,69 @@
+# Cargo
+
+<https://doc.rust-lang.org/cargo/>
+
+## Prerequisites
+
+To use Cachi2 with Cargo locally, ensure you have Cargo installed on your system.
+The [recommended](https://www.rust-lang.org/tools/install) way to install Cargo
+along with Rust toolchain is to use the `rustup` tool.
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Then, ensure that the **Cargo.toml** and **Cargo.lock** are in your project directory.
+
+## Basic usage
+
+Run the following commands in your terminal to pre-fetch your project's
+dependencies specified in the **Cargo.lock**. It must be synchronized with the **Cargo.toml**
+file. Otherwise, the command will fail.
+
+```bash
+cd path-to-your-rust-project
+cachi2 fetch-deps cargo
+```
+
+The default output directory is `cachi2-output`. You can change it by passing
+the `--output-dir` option for the `fetch-deps` command.
+
+In addition, the command will update the `.cargo/config.toml` to use the vendored
```

Comment:
This is the first time the word `vendored` is mentioned in this context. It may be confusing to the user to realize at this point what we're doing. I think this whole paragraph deserves a bit of a careful rewrite on how our fetching works and what it means for the user (i.e. the need to update their cargo config file).